src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -320,9 +321,21 @@ export default class Browser extends DashboardView {
     return data;
   }
 
+  async fetchParseDataCount(source, filters) {
+    const query = queryFromFilters(source, filters);
+    const count = await query.count({ useMasterKey: true });
```

Comment:
Potential performance issue - if a filter uses non-indexed fields then count() will perform a colscan. This could also happen in fetchParseData so may not be a concern, however count has no option to early exit by hitting a limit whereas fetchParseData does.

I'm not sure how to solve this, perhaps have to rely on parse-server having databaseOptions.maxTimeMS specified?